### PR TITLE
chore: export GroupedListV2 types

### PR DESCRIPTION
**Pull request by spmonahan**
## Previous Behavior

Some types for `GroupedListV2` are not exported.

## New Behavior

`GroupedListV2` props and state types are exported.

### CODE REVIEW

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/e49b13e82eddc73cbc860ec0822afb916c679a7e'>e49b13e82eddc73cbc860ec0822afb916c679a7e</a></sub>
<div id="bundleSizeComment9423"/>

File: packages/react/src/components/GroupedList/GroupedListV2.tsx [hunk 1]
```diff
@@ -25,3 +25,4 @@ const GroupedListV2: React.FunctionComponent<IGroupedListV2Props> = styled<
 GroupedListV2.displayName = 'GroupedListV2_unstable';
 
 export { GroupedListV2 as GroupedListV2_unstable };
+export type { IGroupedListV2Props } from './GroupedListV2.types';
```
Comment (khmakoto) [inline packages/react/src/components/GroupedList/GroupedListV2.tsx:28]: Do you need to export it here if you're already exporting everything from `GroupedListV2.types` in the `index` file below?

**Comment by fabricteam** [comment]
## [🕵 fluentuiv8 Open the Visual Regressions report to inspect the 1 screenshots]( https://staticpagerelease.z16.web.core.windows.net/azurestaticapps/vr-fluent/index.html?organization=uifabric&project=fabricpublic&prId=26052&commitId=2d549769a2d08a0c71e8f6803bb03bef462ab645&env=prod)

✅ There was 0 screenshots added, 0 screenshots removed, 1044 screenshots unchanged, 0 screenshots with different dimensions and 1 screenshots with visible difference.


  <details><summary><b>unknown</b> 1 screenshots</summary>

|Image Name|Diff(in Pixels)| Image Type
|-|-|-|
|Coachmark.Collapsed.default.chromium.png|159| Changed|

</details>


<div id="vrtCommentfluentuiv8"/>

File: packages/react/src/components/GroupedList/GroupedListV2.tsx (hunk 1 above)
Comment (spmonahan) [reply packages/react/src/components/GroupedList/GroupedListV2.tsx:28]: Probably not. However, I mirrored what is exported in `[GroupedList](https://github.com/microsoft/fluentui/blob/master/packages/react/src/components/GroupedList/GroupedList.ts#L15)` so I'm leaning toward having the export here but don't feel very strongly about it.

**Review by khmakoto — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/BaseButton-mount_base.html">1424</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/BaseButton-mount.html">1416</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Breadcrumb-mount_base.html">3489</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Breadcrumb-mount.html">3478</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Checkbox-mount_base.html">3150</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Checkbox-mount.html">3138</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/CheckboxBase-mount_base.html">2797</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/CheckboxBase-mount.html">2795</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ChoiceGroup-mount_base.html">5313</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ChoiceGroup-mount.html">5284</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ComboBox-mount_base.html">1473</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ComboBox-mount.html">1531</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/CommandBar-mount_base.html">11340</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/CommandBar-mount.html">11220</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ContextualMenu-mount_base.html">12773</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ContextualMenu-mount.html">12782</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DefaultButton-mount_base.html">1655</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DefaultButton-mount.html">1665</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DetailsRow-mount_base.html">4297</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DetailsRow-mount.html">4256</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DetailsRowFast-mount_base.html">4278</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DetailsRowFast-mount.html">4324</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DetailsRowNoStyles-mount_base.html">4083</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DetailsRowNoStyles-mount.html">4104</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Dialog-mount_base.html">3615</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Dialog-mount.html">3650</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DocumentCardTitle-mount_base.html">682</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/DocumentCardTitle-mount.html">662</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Dropdown-mount_base.html">3786</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Dropdown-mount.html">3840</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/FocusTrapZone-mount_base.html">2401</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/FocusTrapZone-mount.html">2384</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/FocusZone-mount_base.html">2356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/FocusZone-mount.html">2325</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedList-mount_base.html">2233</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedList-mount.html">2528</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedList-virtual-rerender_base.html">1356</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedList-virtual-rerender.html">1331</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">1964</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedList-virtual-rerender-with-unmount.html">1957</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedListV2-mount_base.html">679</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedListV2-mount.html">664</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedListV2-virtual-rerender_base.html">635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedListV2-virtual-rerender.html">655</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">653</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">681</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/IconButton-mount_base.html">2218</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/IconButton-mount.html">2237</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Label-mount_base.html">870</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Label-mount.html">882</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Layer-mount_base.html">5111</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Layer-mount.html">5209</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Link-mount_base.html">975</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Link-mount.html">962</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/MenuButton-mount_base.html">1987</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/MenuButton-mount.html">1964</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/MessageBar-mount_base.html">2772</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/MessageBar-mount.html">2755</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Nav-mount_base.html">3883</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Nav-mount.html">3867</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/OverflowSet-mount_base.html">1635</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/OverflowSet-mount.html">1612</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Panel-mount_base.html">2998</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Panel-mount.html">3038</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Persona-mount_base.html">1569</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Persona-mount.html">1544</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Pivot-mount_base.html">1926</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Pivot-mount.html">1963</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/PrimaryButton-mount_base.html">1796</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/PrimaryButton-mount.html">1778</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Rating-mount_base.html">8240</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Rating-mount.html">8261</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/SearchBox-mount_base.html">1751</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/SearchBox-mount.html">1797</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Shimmer-mount_base.html">3368</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Shimmer-mount.html">3408</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Slider-mount_base.html">2503</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Slider-mount.html">2498</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/SpinButton-mount_base.html">5543</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/SpinButton-mount.html">5511</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Spinner-mount_base.html">962</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Spinner-mount.html">940</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/SplitButton-mount_base.html">3630</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/SplitButton-mount.html">3693</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Stack-mount_base.html">984</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Stack-mount.html">1002</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/StackWithIntrinsicChildren-mount_base.html">2729</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/StackWithIntrinsicChildren-mount.html">2773</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/StackWithTextChildren-mount_base.html">5623</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/StackWithTextChildren-mount.html">5667</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/SwatchColorPicker-mount_base.html">12148</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/SwatchColorPicker-mount.html">12253</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/TagPicker-mount_base.html">3020</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/TagPicker-mount.html">3013</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TeachingBubble</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/TeachingBubble-mount_base.html">96431</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/TeachingBubble-mount.html">97785</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Text-mount_base.html">937</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Text-mount.html">930</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/TextField-mount_base.html">1890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/TextField-mount.html">1904</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ThemeProvider-mount_base.html">1800</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ThemeProvider-mount.html">1830</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ThemeProvider-virtual-rerender_base.html">1276</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ThemeProvider-virtual-rerender.html">1279</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">2539</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">2529</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Toggle-mount_base.html">1306</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/Toggle-mount.html">1304</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/buttonNative-mount_base.html">643</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/26052/perf-test/buttonNative-mount.html">651</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Pull request merged**